### ISSUE TITLE
[NFR] Implement Model snapshot loading call

### ISSUE THREAD

**Issue by temuri416** [phalcon/cphalcon#11818]
Hi all,

It would be nice to have an API method to allow manual loading of Model's snapshot:

``` php
$robot = new Robot;
$robot->save();
$robot->loadSnapshot();
```

Thanks!

**Comment by temuri416** [comment]
Hi, @sergeyklay. What are your thoughts on this? Should ->save() automatically set updated snapshot?

Thanks!

**Comment by sergeyklay** [comment]
I'll try to sort out as soon as posible

**Comment by Jurigag** [comment]
I made PR with fix for this, let me know what you think.

**Comment by sergeyklay** [comment]
Fixed in the `3.1.x` branch.
